src/draive/agents/visualization.py

```diff
@@ -0,0 +1,150 @@
+from collections.abc import Mapping
+from typing import Any, Literal
+
+from draive.agents.node import AgentMessage
+from draive.parameters import State
+from draive.utils import not_missing
+
+__all__ = [
+    "AgentWorkflowVisualization",
+]
+
+
+class AgentWorkflowVisualization(State):
+    output: Literal["interactive", "console"] | None = None
+
+    def visualize(
```

Comment:
Over 100 lines for plotting function with branching. :ugly:

Reply:
sure, I will rework it yet 😉 I have forgot to set "WIP" status to this PR 😇